### PR TITLE
Added Xunit

### PR DESCRIPTION
**Pull request by hueifeng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ Here are a few examples for throwing your own exception message.
 ```csharp
 if (!ModelState.IsValid)
 {
-    throw new ApiException(ModelState.AllErrors());
+    throw new ApiException(ModelState);
 }
```
Comment (proudmonkey) [inline README.md]: `ApiException` accepts the type of `IEnumerable<ValidationError>` so it should be:

```csharp
throw new ApiException(ModelState.AllErrors());
```

**Comment by proudmonkey** [comment]
This is great. Thanks for the PR. I'll review them :)

File: README.md (hunk 1 above)
Comment (hueifeng) [reply README.md]: I'm sorry I was carelesso (╥﹏╥)o

Comment (proudmonkey) [reply README.md]: PR merged! Thank you! :)

**Comment by GithubUnknownManito** [comment]
整天花里胡哨

**Pull request merged**